framework/platform/bpl/uci/cfg/bpl_cfg_uci.cpp

```diff
@@ -256,6 +278,7 @@ int cfg_uci_get_wireless_idx(char *interfaceName, int *rpc_index)
 
             scanf_res = sscanf_s(s->e.name, "default_radio%d", rpc_index);
 
+#ifdef BEEROCKS_UGW
```

Comment:
I'm not sure this is correct - this used to run for all platforms which used this BPL. So now if RAX40 doesn't have BEEROCKS_UGW defined you will break it - so please change to
```
#ifdef BEEROCKS_RDKB
//DO SOMETHING for RDKB platform
#else 
//DO SOMETHING ELSE for all platforms
#endif
```